cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -147,7 +122,7 @@ def configured_depot(log_depot, depot_machine_ip, appliance):
     It also provides a finalizer to disable the depot after test run.
     """
     log_depot.machine_ip = depot_machine_ip
-    uri = log_depot.machine_ip + log_depot.access_dir
+    uri = urljoin(log_depot.machine_ip, log_depot.access_dir)
```

Comment:
@mshriver this doesn't work as expected and log collection got broken
![image](https://user-images.githubusercontent.com/42433123/58694506-6d60c380-8393-11e9-9ab6-5c1ab6467b18.png)

Reply:
Talked about offline, replacing urljoin with just a format call for now. Discussed better options with @lina-nikiforova, and she's going to rework how these Depot objects are created from yaml.